### ISSUE TITLE
add support for GitLab

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt-cli#122]
What with Microsoft having acquired GitHub, quite a few people have migrated to GitLab. We might consider adding official support for registering plugins hosted on GitLab.

Please 👍 this issue or add a +1 comment below if you think this would be a useful addition....

**Comment by adapter267** [comment]
Adapt, being FOSS, has meant many people can freely contribute all sorts to the growth of this brilliant and important project. There needs to be equal opportunity for developers who don't wish to use GitHub to still be able to contribute key features such as plugins.

**Comment by moloko** [comment]
@adapter267 I agree completely, it's just a case of the project having limited resources to do work such as this...! Particularly at this current moment in time with all the fun of the pandemic going on.

If there's any chance you can spare some time to help out please let us know... we can help get you started.

Updating the adapt-cli shouldn't be too problematic, the trickier issue I suspect will be getting the [plugin browser](https://www.adaptlearning.org/index.php/plugin-browser/) to read data from GitLab.

**Comment by adapter267** [comment]
@moloko I really wish I could help, but unfortunately my programming experience is limited to HTML, CSS, Bash and basic Python. My main areas of expertise are in writing and elearning instructional design, so I'd be happy to contribute to documentation or similar (development, proofreading etc.). 

I use Adapt in my free time for my own courses, and it's a tool we're very invested in where I work (elearning company in Gloucestershire), so I'd love to help out in any way that I can. Particularly as I'm furloughed for the next two weeks.
 
I've included my details below in case I can help in any way.

[samhowell.uk](https://samhowell.uk)
mail@samhowell.uk
